pkg/template/template.go

```diff
@@ -40,20 +41,49 @@ func (p *Processor) Process(template *api.Template) (*configapi.Config, fielderr
 	}
 
 	for i, item := range template.Objects {
+		if obj, ok := item.(*runtime.Unknown); ok {
```

Comment:
I assume we are not reporting the errors to client, just reporting them to a server log, correct?